scripts/source/02-make_forward.py

```diff
@@ -87,9 +87,18 @@ def run_forward(subject, session=None):
     else:
         conductivity = (0.3,)
 
-    bem_model = mne.make_bem_model(subject=fs_subject,
-                                   subjects_dir=fs_subjects_dir,
-                                   ico=4, conductivity=conductivity)
+    try:
+        bem_model = mne.make_bem_model(subject=fs_subject,
+                                       subjects_dir=fs_subjects_dir,
+                                       ico=4, conductivity=conductivity)
+    except FileNotFoundError as exc:
+        message = ("Could not make BEM model due to a missing file. \n"
+                   "Can be solved by setting recreate_bem=True in the config "
+                   "to force recreation of the BEM model, or by deleting the\n"
+                   f" {config.bids_root}/derivatives/freesurfer/subjects/"
+                   f"sub-{subject}/bem/ folder")
+        raise FileNotFoundError(message) from exc
```

Comment:
Why do you do an explicit exception chaining here? I think we can easily do without it, no?

Reply:
If I don't, the exceptions are chained with the message `During handling of the above exception, another exception occurred:`. Currently it is `The above exception was the direct cause of the following exception:` which I thought was better.

Reply:
I don't know the best practice though so I should probably remove it?

Reply:
I would remove it for now.

I'm super open towards discussing our entire logging and exception handling system in a separate issue / PR!

Reply:
what trigged your motivation for this PR? just curious

Reply:
It's easy to get a crash if something went wrong when creating the BEM model, and there was no indication on what to do, so I expected newcomers to have trouble with it.
I opened issue #304 related to that, and I proposed to open a PR.

Reply:
@MerlinDumeur Can you remove the explicit exception chaining for now?